src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.SslProvider.cs

```diff
@@ -95,7 +95,12 @@ internal static void SetSslOptions(EasyRequest easy, ClientCertificateOption cli
             private static void SetSslVersion(EasyRequest easy, IntPtr sslCtx = default(IntPtr))
             {
                 // Get the requested protocols.
-                System.Security.Authentication.SslProtocols protocols = easy._handler.ActualSslProtocols;
+                SslProtocols protocols = easy._handler.SslProtocols;
```

Comment:
I don't understand the question.  Are you asking how it's accessing easy._handler?  _handler is internal, not private.